admin/admin.php

```diff
@@ -24,18 +24,14 @@
  * filters_post    => reference to PLL_Admin_Filters_Post object
  * filters_term    => reference to PLL_Admin_filters_Term object
  * nav_menu        => reference to PLL_Admin_Nav_Menu object
- * sync            => reference to PLL_Admin_Sync object
  * block_editor    => reference to PLL_Admin_Block_Editor object
  * classic_editor  => reference to PLL_Admin_Classic_Editor object
  * filters_media   => optional, reference to PLL_Admin_Filters_Media object
- * wizard          => reference, a PLL_Wizard object
  *
  * @since 1.2
- * @since 2.7 Added a reference to a PLL_Bulk_Translate instance.
- * @since 2.7 Added a reference to a PLL_Wizard object.
  */
 class PLL_Admin extends PLL_Admin_Base {
-	public $filters, $filters_columns, $filters_post, $filters_term, $nav_menu, $sync, $filters_media;
```

Comment:
This will need to be reformatted to be parsed by PHPDocumentor.

Reply:
I remember that you opened a separate issue for this: https://github.com/polylang/polylang-pro/issues/544